lr_face/data_providers.py

```diff
@@ -89,17 +90,38 @@ def enfsi_data(resolution, year) -> PairsWithIds:
                 questioned_ref = file[0]
             else:
                 raise ValueError(f'Unknown ENFSI year {year}')
-            y[cls-1] = int(df.loc[cls]['same'] == 1)
-            ids[cls-1] = f"enfsi_{year}_{cls}_{int(df.loc[cls]['same'] == 1)}"
+            y[cls - 1] = int(df.loc[cls]['same'] == 1)
+            ids[cls - 1] = f"enfsi_{year}_{cls}_{int(df.loc[cls]['same'] == 1)}"
             if questioned_ref == 'q':
-                X[cls-1][0] = img
+                X[cls - 1][0] = img
             elif questioned_ref == 'r':
-                X[cls-1][1] = img
+                X[cls - 1][1] = img
             else:
                 raise ValueError(f'unknown questioned/ref: {questioned_ref}')
     return PairsWithIds(pairs=X, is_same_source=y, pair_ids=ids)
 
 
+def forenface_data(resolution) -> ImageWithIds:
+    folder = os.path.join("resources", "forenface")
+    files = os.listdir(folder)
+
+    # only take randomly selected items from forenface, to keep it small
+    # todo: make max_files configurable in params.py
+    max_files = 200
+    random_ids = random.sample(range(len(files)), max_files)
+    downsized_files = [j for i, j in enumerate(files) if i in random_ids]
+
+    images = []
+    person_ids = []
+    image_ids = list(range(max_files))
```

Comment:
niet informatiever  hier de bestandsnamen te gebruiken?

Reply:
of forenface+bestandsnaam

Reply:
Ja, goed idee. Toen ik dit typte zat ik nog in m'n hoofd met, waarom is dit niet gewoon een int.